### PR TITLE
fix(building-webpack): allow to provide a publicPath

### PR DESCRIPTION
**Pull request by paales**

### CODE REVIEW

**Comment by LarsDenBakker** [comment]
Thanks for your contribution. I think the change is ok. What problem are you solving with this?

**Comment by daKmoR** [comment]
quick question why a separate parameter and not part of the options object?

**Comment by paales** [comment]
> quick question why a separate parameter and not part of the options object?

Not really into the webpack conventions so I was in doubt if I should create `option.publicPath` or access `compiler.options.output.publicPath`. What would you suggest @daKmoR?

**Comment by paales** [comment]
> Thanks for your contribution. I think the change is ok. What problem are you solving with this?

We're actually integrating the open-wc stuff into an existing project which requires different paths when the website is actually served.

File: packages/building-webpack/src/modern-web-webpack-plugin.js [hunk 1]
```diff
@@ -80,14 +66,27 @@ function createEntryVariable(entries, development) {
     : ${asArrayLiteral(entries.legacy)};`;
 }
 
-function createPolyfillsLoaders(options) {
+function createPolyfillsLoaders(options, publicPath) {
```
Comment (paales) [inline packages/building-webpack/src/modern-web-webpack-plugin.js]: I should probably rename this to `polyfillsPublicDir` makes more sense.

**Comment by LarsDenBakker** [comment]
Ah, that makes perfect sense to add in. 

@daKmoR the separate parameter is because it's not a user provided option, it's an option retreived inside the plugin from the webpack config.

Good to go, you just need to run `npm run format` from the repository global to format your files.

**Comment by paales** [comment]
@LarsDenBakker I'm unable to do so, if I checout the project run npm install and then npm run format I get: `ESLint couldn't find the plugin "eslint-plugin-html".`.
After adding that one: Error: Cannot find module 'babel-eslint'

Not sure what you are expecting, maybe you can run it your self?

**Comment by daKmoR** [comment]
@paales I will do it :hugs: 

can you sign the CLA then I will make sure to release it?

**Comment by paales** [comment]
@daKmoR Signed :)

**Comment by daKmoR** [comment]
thank you for your contribution ❤️ 

PS: sorry for the delay 🙈  better late then never right 😅 🤣

**Pull request merged**